### DISCUSSION TITLE
news and interests(widgets) still running in background after disabling

### DISCUSSION THREAD

**Discussion by biskyy** [Atlas-OS/Atlas#1262] [Help & Support]
windows 10, ver 0.4

news and interests(widgets) are still running in background even after disabling it.

![image](https://github.com/user-attachments/assets/f8c7dac5-8ab3-4d2b-9563-c693b732bb38)
![image](https://github.com/user-attachments/assets/724574c6-8a62-494e-8bdf-236c155605af)

as you can see it sometimes takes ~15mb of ram and sometimes ~80mb. it's very random.

**Comment by overwatch3560** [comment]
@biskyy try downloading 24h2 with version 0.4.1 for me it was fixed in the new version

**Reply by biskyy** [reply]
what..? no
